iosApp/iosAppTests/iosAppTests.swift

```diff
@@ -101,6 +101,40 @@ class iosAppTests: XCTestCase {
         contentController.disconnectMessenger()
     }
 
+    func testConnectionClosed() {
+        // Pull the deployment for use later.
+        var deployment: Deployment?
+        do {
+            deployment = try Deployment()
+        } catch {
+            XCTFail("\(error.localizedDescription)")
+        }
+        guard let deployment else {
```

Comment:
I love this new Swift feature making a shorthand for shadowing optionals that already exist, but our Jenkins agent isn't up to date yet so I fear this will cause it to complain.

Reply:
Good point. I'll update the test to make sure an error doesn't happen.